server/tools/sql_tables/Training.sql

```diff
@@ -0,0 +1,23 @@
+SET FOREIGN_KEY_CHECKS = 0;
+--
+-- Database: `cAuth`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `Training`
+--
+DROP TABLE IF EXISTS `Training`;
+CREATE TABLE `Training` (
+  `Location` varchar(255) NOT NULL,
+  `LocationDetail` varchar(255) NOT NULL,
+  `Day` varchar(255) NOT NULL,
+  `Time` time NOT NULL,
```

Comment:
can we use DATETIME type for both day and time? and we probably want to add an index on this column.